source/common/http/http1/codec_impl.cc

```diff
@@ -742,7 +742,8 @@ StatusOr<ParserStatus> ConnectionImpl::onHeadersComplete() {
   // CONNECT request has no defined semantics, and may be rejected.
   if (request_or_response_headers.TransferEncoding()) {
     const absl::string_view encoding = request_or_response_headers.getTransferEncodingValue();
-    if (!absl::EqualsIgnoreCase(encoding, header_values.TransferEncodingValues.Chunked) ||
+    if (!HeaderUtility::headerValueContainsIgnoreCase(
```

Comment:
Along with handling chunked, chunked, this brings Envoy from rejecting unknown transfer encodings to passing them though.  I don't think we should allow that by default. 
If your intent is just to allow multiple chunked headers and we decide to go that route, let's be sure there's nothing but "identity" and "chunked" in the header using StringUtil::splitToken and removing whitespace.  If you really want to allow everything through we'll want it config guarded and off by default.

Reply:
+1

we shouldn't change default behavior.